x-pack/docs/en/security/authentication/oidc-guide.asciidoc

```diff
@@ -381,8 +381,8 @@ the user back to after the OP logs them out.
 
 When configuring `rp.post_logout_redirect_uri`, care should be taken to not point this to a URL that
 will trigger re-authentication of the user. For instance, when using OpenID Connect to support
-single sign-on to {kib}, this could be set to +$\{kibana-url}/logged_out+,  which will show a user-
-friendly message to the user.
+single sign-on to {kib}, this could be set to either +$\{kibana-url}/security/logged_out+,  which will show a user-
+friendly message to the user or +$\{kibana-url}/login?msg=LOGGED_OUT+ which will take the user to the login selector in {kib}.
```

Comment:
There's an extra space showing up between "user-" and "friendly", so I'd recommend putting them on the same line:
```suggestion
user-friendly message to the user or +$\{kibana-url}/login?msg=LOGGED_OUT+ which will take the user to the login selector in {kib}.
```